### PR TITLE
Update versioning policy to include 4.x examples for 4.1

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1749121

### CODE REVIEW

File: release_notes/versioning-policy.adoc [hunk 1]
```diff
@@ -21,10 +21,10 @@ cannot use it to complete all of the commands in {product-title}
 {product-version}. You must download and install the new version of `oc`.
 
 Changes of APIs for non-security related reasons will involve, at minimum, two
-minor releases (3.4 to 3.5 to 3.6, for example) to allow older `oc` to update.
-Using new capabilities may require newer `oc`. A 3.2 server may have additional
-capabilities that a 3.1 `oc` cannot use and a 3.2 `oc` may have additional
-capabilities that are not supported by a 3.1 server.
+minor releases (3.10 to 3.11 to 4.1, for example) to allow older `oc` to update.
```
Comment (vikram-redhat) [inline release_notes/versioning-policy.adoc]: @sferich888 is it safe to say here in our version policy guideline to mix 3.x and 4.x releases?

Comment (sferich888) [reply release_notes/versioning-policy.adoc]: I think it can, from a cli perspective, but this is more a question for @deads2k

**Comment by mburke5678** [comment]
@zhouying7780 Please take a look. Thank you!

**Comment by zhouying7780** [comment]
LGTM。

**Pull request merged**